edge-hub/test/Microsoft.Azure.Devices.Edge.Hub.E2E.Test/EdgeToDeviceMethodTest.cs

```diff
@@ -19,10 +20,10 @@ public class EdgeToDeviceMethodTest : IClassFixture<ProtocolHeadFixture>
         public async Task InvokeMethodOnModuleTest(ITransportSettings[] transportSettings)
         {
             // Arrange
+            string deviceName = string.Format("moduleMethodTest-{0}", transportSettings.First().GetTransportType().ToString("g"));
```

Comment:
If we are creating a device, we should add a uuid to it, otherwise 2 tests running at the same time might clash.